beetsplug/smartplaylist.py

```diff
@@ -201,6 +210,7 @@ def update_playlists(self, lib):
                 if item_path not in m3us[m3u_name]:
                     m3us[m3u_name].append(item_path)
 
+        started_with = bytestring_path(self.config['started_with'].get())
```

Comment:
I don't think `bytestring_path` is the right thing here because this isn't necessarily a path. I think we should encode it with whatever the text encoding is for the playlist file?